adminutils/adminutils.py

```diff
@@ -263,6 +281,7 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
             )
         except discord.Forbidden:
             await ctx.send(chat.error(_("I can't edit this emoji")))
+        await ctx.send(_(f"{em} edited to `{em.name}`."))
         await ctx.tick()
```

Comment:
```suggestion
        else:
            await ctx.send(_(f"{em} edited to `{em.name}`."))
            await ctx.tick()
```
Otherwise it will fail, since `em` will be not defined on exception